packages/reporter/src/commands/object-viewer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import _ from 'lodash'
+import Collapsible from '../collapsible/collapsible'
+
+interface ObjectViewerProps {
+  obj: Record<any, any>
+  isOpen: boolean
+}
+
+export const ObjectViewer = ({ obj, isOpen }: ObjectViewerProps) => {
+  return Object.keys(obj).length === 1 && !_.isObject(obj[Object.keys(obj)[0]])
+    ? (
+      <div className="object-viewer">
+        {`{ ${Object.keys(obj).map((key) => `${key}: ${encode(obj[key])}`).join('')} }`}
+      </div>
+    )
+    : (
+      <Collapsible
+        header={{
+          collapsed: 'Show options',
+          expanded: 'Hide options',
+        }}
+        isOpen={isOpen}
+      >
+        <pre>{JSON.stringify(obj, null, 4)}</pre>
```

Comment:
We use prismjs to highlight code (in the error code frame for example). I think this would be nice to add to this component for easier readability. I think some more work needs to be done to include this.

```suggestion
        <pre>
          <code className='language-json'>{JSON.stringify(obj, null, 4)}</code>
        </pre>
```

Reply:
**Before:**
![Screenshot from 2020-08-27 18-37-41](https://user-images.githubusercontent.com/8130013/91424321-9e7cbc00-e894-11ea-8ff4-19ab9d2829a9.png)

**After:**
![Screenshot from 2020-08-27 18-38-21](https://user-images.githubusercontent.com/8130013/91424331-a177ac80-e894-11ea-963d-d932da279202.png)

I think before is darker and more legible. Is it only me?

Reply:
Yah, I'll have to add some nicer styles for JSON specifically. We just didn't import the styles because we haven't used JSON anywhere in the project before.